spec.emu

```diff
@@ -276,6 +276,95 @@
       <emu-note>That means that "A" (`LATIN CAPITAL LETTER A`) measures as 1 code unit, and "🔥" (`FIRE`) measures as 2 code units. Source maps for other content types may diverge from this.</emu-note>
     </dd>
   </dl>
+
+  <emu-clause id="sec-vlq-grammar">
```

Comment:
Ideally "terms and definitions" should have "simple" definitions, easily expressible as "name -> meaning", like for a human-language dictionary.

I would prefer to move this emu-clause to the top level, remove the existing _base64 VLQ_ `<dfn>` from "terms and definitions", and use it as an introductory paragraph for this new section, including the examples.

Reply:
Done, I renamed the top level section to "base64 VLQ".